react-client/src/Components/App.js

```diff
@@ -48,7 +48,7 @@ function App() {
 
         const children = [...e.target.children].map((child) => {
           return {
-            id: child.id.trim() !== "" ? "#" + child.id : null,  class: child.className.trim() !== "" ? "." + child.className : null,
+            id: child.id.trim() !== "" ? "#" + child.id : null,  class: child.className.trim() !== "" ? "." + child.className : null, childtag: child.localName
```

Comment:
You can use string interpolation like in this example:
id: child.id.trim() ? `#${child.id}` : null

Reply:
`childtag` label is redundant I think. You can change it to `tag` instead

Reply:
noted already done